lib/views/pages/events/add_event_page.dart

```diff
@@ -140,7 +132,9 @@ class _AddEventState extends State<AddEvent> {
     );
     print('Result is : $result');
     if (result == null) {
-      _exceptionToast("Could not create event! Please Try Again later!");
+      Fluttertoast.showToast(
+          msg: "Could not create event! Please Try Again later!",
```

Comment:
no hardcoding of strings. Internationalization file please.

Reply:
This is going to be already big a seperate issue for that would be much better is guess.